src/specs/NativeA0Auth0.ts

```
@@ -23,7 +23,8 @@ export interface Spec extends TurboModule {
     clientId: string,
     domain: string,
     localAuthenticationOptions:
-      { [key: string]: string | Int32 | boolean } | undefined,
+      | { [key: string]: string | Int32 | boolean }
+      | undefined,
```

ESLint

[error] Replace `|·{·[key:·string]:·string·|·Int32·|·boolean·}⏎·····` with `{·[key:·string]:·string·|·Int32·|·boolean·}`

(prettier/prettier)